packages/logging/src/analytics-helpers.ts

```diff
@@ -112,12 +121,12 @@ export class ToggleableAnalytics implements MongoshAnalytics {
 
   identify(...args: Parameters<MongoshAnalytics['identify']>): void {
     this._validateArgs(args);
-    this._queue.push(['identify', args]);
+    this._queue.push(['identify', [addTimestamp(args[0])]]);
```

Comment:
TIcket only mentioned `ThrottledTelemetry`, I though we'd want both queuing telemetries to preserve the timestamp to the time of issue from the userland?

Reply:
Yes :)